readme.md

```diff
@@ -1,78 +1,36 @@
+
 <p align="right">
 <a href="https://autorelease.general.dmz.palantir.tech/palantir/spark-tpcds-benchmark"><img src="https://img.shields.io/badge/Perform%20an-Autorelease-success.svg" alt="Autorelease"></a>
 </p>
 
-TPCDS Spark Benchmark Runner
-==================================
+Spark Benchmark Runner
+======================
+
+This repo contains tools to run 2 industry standard benchmark suites:
 
-TPC-DS is an open benchmark suite for structured data systems. This utility aims to make it easy to generate TPC-DS
-data, and to run TPC-DS benchmarks against different versions of Spark. The main use case for this utility is to run
-scale tests when developing Spark itself. For example, one can use this tool to check the performance of a change made
-to the way Spark shuffle works.
+1. [TPC-DS](http://www.tpc.org/tpcds/) is an open benchmark suite for structured data systems. This utility aims to make it easy to generate TPC-DS data, and to run TPC-DS benchmarks against different versions of Spark. The main use case for this utility is to run scale tests when developing Spark itself. For example, one can use this tool to check the performance of a change made to the way Spark shuffle works. The benchmarks run SQL queries against structured datasets. This utility is thus not useful for running tests in streaming workflows.
 
-The benchmarks run SQL queries against structured datasets. This utility is thus not useful for running tests in
-streaming workflows.
+2. [Sort Benchmark](http://sortbenchmark.org/): This is a single benchmark that sorts a large amount of data generated by the [gensort](http://www.ordinal.com/gensort.html) program.
 
 The benchmark suite can be run on MacOS or CentOS 6+. It does not currently support running on Windows.
 
 # Usage
 
-The benchmark suite requires a distributed storage layer to put the generated test data as well as the computation
-results. This tool also requires a cluster manager for running the Spark driver and executors.
+The benchmark suite requires a distributed storage layer (such as HDFS/S3/Azure Blob Storage) to put the generated test data, as well as the computation
```

Comment:
`s/"to put"/"to store"/g`

Also maybe we should change this whole sentence to make it explicit that it's not actually required? E.g., you can run in local mode against a local filesystem for small scale / testing.